scripts/globals/magic.lua

```diff
@@ -1130,6 +1130,14 @@ function doDivineNuke(V,M,caster,spell,target,hasMultipleTargetReduction,resistB
 end
 
 function doNinjutsuNuke(V,M,caster,spell,target,hasMultipleTargetReduction,resistBonus,mabBonus)
+    if (mabBonus == nil) then
```

Comment:
can also ``mabBonus = mabBonus or 0;`` here instead (you dont need to change this if you dont want to, just for future reference)